Doc/whatsnew/3.11.rst

```diff
@@ -739,11 +745,34 @@ For major changes, see :ref:`new-feat-related-type-hints-311`.
   to clear all registered overloads of a function.
   (Contributed by Jelle Zijlstra in :gh:`89263`.)
 
-* :data:`typing.TypedDict` subclasses can now be generic. (Contributed by
-  Samodya Abey in :gh:`89026`.)
+* The :meth:`__init__` method of :class:`~typing.Protocol` subclasses
+  is now preserved. (Contributed by Adrian Garcia Badarasco in :gh:`88970`.)
 
-* :class:`~typing.NamedTuple` subclasses can now be generic.
-  (Contributed by Serhiy Storchaka in :issue:`43923`.)
+* The representation of empty tuple types (``Tuple[()]``) is simplified.
+  (Contributed by Serhyi Storchaka in :gh:`91137`.)
+
+* Loosen runtime requirements for type annotations by removing the callable
+  check in the private ``typing._type_check`` function. (Contributed by
+  Gregory Beauregard in :gh:`90802`.)
```

Comment:
Should we really include this? It's a private undocumented function (though David's recent bug report suggested to me otherwise).

Reply:
Given that it's already spawned a bug report, it's likely that other users will run into this change, so I think it's important to include.

Reply:
Agreed then. Could you please add a notice strongly discouraging users from using it too? We might as well try to dissuade users though it might be in vain :).
```rst
This function is for internal use only and is **unstable** and may change across versions. Please do not use this function. Instead, we recommend implementing this with your own code.
```